### ISSUE TITLE
does not show Favorites apps Fedora 30

### ISSUE THREAD

**Issue by Bezmehrabi** [micheleg/dash-to-dock#941]
hi I installed Fedora30 and now dash to dock not showing my Favorites apps

![Screenshot from 2019-04-30 20-59-03](https://user-images.githubusercontent.com/46019949/56977640-e13c6000-6b8a-11e9-8c9b-a42cec700061.png)

![Screenshot from 2019-04-30 20-58-40](https://user-images.githubusercontent.com/46019949/56977624-d71a6180-6b8a-11e9-96ff-05437eb46980.png)

**Comment by luizfernando-outcenter** [comment]
On my case, I had an update pending of the extension. After update, I just restart my computer. Everything is working fine now.

**Comment by Findarato** [comment]
I had that problem too. After removing and updating it seems to be working ... ish

**Comment by rugk** [comment]
![image](https://user-images.githubusercontent.com/11966684/56990976-d7205e80-6b96-11e9-9492-581f1bc06d61.png)

directly after update…

**Comment by rugk** [comment]
Note it also did not _running_ apps, basically it showed only the "all applications" button.

**Comment by agedz** [comment]
first you need to remove extemsion, then remove folder of extension in .local/share/gnome-shell/extensions
after install extension again and it will work normal

**Comment by Bezmehrabi** [comment]
just restart and it's working for me. thanks.

**Comment by rugk** [comment]
Well… but it's still a bug IMHO. I would keep this open…

**Comment by chetankhilosiya** [comment]
The same issue has happened with me, I would like to help with debugging the issue. Anything I can do?

**Comment by harryhobson** [comment]
I recently upgraded to Fedora 30. Similar issues:
1. only see the 'All applications' dots (no favourites)
1. cannot see any open applications
1. have trouble activating mouseover behaviour of Integllient autohide (dock doesn't appear)

**Comment by unfulvio** [comment]
I have the same issue, just updated from 29.

Restarting didn't help. Uninstalling extension, then restarting, then resintalling didn't work either.

Neither favorites or currently active programs are showing, only the dots icon to open the all applications search.

**Comment by OmarHussainX** [comment]
> **Edit - issues seem to have disappeared after installing latest Software updates (included newer Kernel)**
> I recently upgraded to Fedora 30. Similar issues:
> 
> 1. only see the 'All applications' dots (no favourites)
> 2. cannot see any open applications
> 3. have trouble activating mouseover behaviour of Integllient autohide (dock doesn't appear)

Can confirm all three of these issues after upgrading to Pop!_OS 19.04. Have tried uninstalling & reinstalling the extension to no avail.

**Comment by stockphrase** [comment]
I suddenly have this same problem under Ubuntu 19.04.

**Comment by tsukirockstar** [comment]
This only happens to me using wayland and starting my session with an usb either stick or external disk.
If you start your session with X it will probably not happen assuming that somehow this can be reproduced on different hard.

Also: This still present on fedora 32 with an up to date version of the extension and I noticed that it not starts only with the USB itself, but after installing OpenCL for my radeon card.

Try with an X session and comment. it would not be the first wayland painful weirdness..

**Comment by Jackbaude** [comment]
I too need this fixed in fedora 32

**Comment by tsukirockstar** [comment]
@Jackbaude As I wrote, seems to be related to wayland and some USB peripherals. Try with X and the dash issue will be solved. However, since the latest updates I don't have any issue, but yet again, I'm not using the default wayland session. Cheers.

**Comment by rugk** [comment]
...and as said, restarting/rebooting fixes this. Still annoying however, you're right.

**Comment by tsukirockstar** [comment]
@rugk Yes. Is annoying. specially if you plug in a gamepad to use with retroarch and forget about the issue... 
Another thing that I noticed (at least on my particular case) is that if you plug in a Wacom or sometimes a wireless gamepad you will get no updates while the peripherals are plugged in. That needs to be reproduced. If anyone else experience that, it needs to be fixed. Is a nasty bug.

**Comment by kbucheli** [comment]
When I login with my mobile phone attached (USB C), then the favorites are missing.
Do I log in with mobile phone detached, then the favorites are there.


RHEL 9.5 with
- `gnome-shell-40.10-22.el9_5.x86_64`
- `gnome-shell-extension-dash-to-dock-40.7-19.el9.noarch`
